certora/harnesses/SocialRecoveryModuleHarness.sol

```diff
@@ -0,0 +1,8 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity >=0.8.12 <0.9.0;
+import {SocialRecoveryModule} from "../../contracts/modules/social_recovery/SocialRecoveryModule.sol";
+import {IGuardianStorage} from "../../contracts/modules/social_recovery/storage/IGuardianStorage.sol";
+
+contract SocialRecoveryModuleHarness is SocialRecoveryModule {
```

Comment:
Why is this harness contract needed?

Reply:
This will be used in the upcoming PRs. As we updated the Harness for the `GuardianStorage` in this PR, thought to add this as well.